competitive-programming-3.py

```diff
@@ -0,0 +1,75 @@
+#Pascal Triangle II: We just want the nth row of the pascal triangle
```

Comment:
You attempted a follow-up question instead of https://leetcode.com/problems/pascals-triangle/ , but both of them are similar